emoji/emoji_transfer_service.py

```diff
@@ -0,0 +1,20 @@
+from emoji import image_client
+
+def transfer(source_client, destination_client, emoji_name):
+    source_dict = source_client.emoji_dict()
+    destination_dict = destination_client.emoji_dict()
+
+    emoji_path = source_dict.get(emoji_name)
+
+    if emoji_path.startswith("alias:"):
+        """If it's an alias, we want to get the url for the file"""
+        aliased_from = emoji_path.replace("alias:", "")
+        emoji_path = source_dict.get(aliased_from)
+
+    if emoji_path is None:
+        """Making an alias for a standard emoji"""
+        return destination_client.add_alias(emoji_name, aliased_from)
```

Comment:
i don't get the flow here (don't know python). it reads like the alias (above) is looking for the url to copy it.

...but then how does aliased_from get set if emoji_path is empty?

Reply:
so the dict looks like
```
{ "emoji-name": "alias:original-emoji" }
```

if `original-emoji` isn't in the dict somewhere (i.e. it's one of the non-custom ones), then `emoji_path` is `None` but `emoji_name` is still `original-emoji`